### DISCUSSION TITLE
NCIRF PSD output variability

### DISCUSSION THREAD

**Discussion by mathomas1481** [ncidose/ncidosetools#30] [NCIRF] [answered]
Choonsik,

We have noticed that there is a clear trend with PSD output values using version 2.0.20240301 (Windows).

The PSD estimate decreases as # of histories increases. Of course, the error associated with any estimate will be higher with fewer histories. But I would think that error would be random, not biased. This appears to show that all the error at lower histories produces a higher PSD estimate.

Is this a bug with the code? Can you comment on what number of histories is producing the most accurate PSD estimate? It seems that if the # of histories were to continue to increase, the PSD estimates would just keep decreasing.

Thanks,

----------------------------
Allan Thomas, PhD, DABR
Assistant Professor of Radiology
Mallinckrodt Institute of Radiology   |   Washington University School of Medicine

**Comment by haeginh** [comment]
Dear Dr. Allan Thomas,

Hi, I am Haegin Han, a PostDoc fellow responsible for the PSD calculations at NCIRF.

Thank you for bringing this issue to our attention. The current version of the NCIRF (2.0.20240301) calculates the PSD by selecting the skin voxel with the highest dose. While this straightforward method appears logical, it inherently leads to higher PSD values as the number of histories decreases. This occurs because, particularly for large beam areas, numerous voxels will have similar dose amounts, and the program selects the voxel with the highest dose among them. Consequently, this selection process introduces a bias, as you correctly pointed out.

We are aware of this issue and are currently developing a post-processing algorithm to address it. This algorithm is still under demonstration and may be incorporated into the next version of the NCIRF.

To obtain a reasonable PSD value with the current version of the NCIRF, we recommend achieving an error margin of 1%.

Thank you for your report.

Best regards,  
Haegin Han

**Comment by mathomas1481** [comment]
Thanks for the response and information, Haegin. Choonsik had forwarded
your response to this question to me earlier in the day as well. He wanted
me to add the question to the NCI Forum so that others could see it as well.

One more detailed question for you: in your response to Choonsik you
mentioned an empirical relationship between the PSD error reported by NCIRF
and the actual error that would result for the PSD output values.

Let’s say the PSD error according to the NCIRF output is 5%, or 0.05. Does
that mean the actual PSD value is roughly overestimated by (15*0.05)^1.6 =
0.63 or 63%? Am I understanding how you set up that formula correctly?


I did not want to put this question on the NCI Forum in case you all were
not ready to disclose the estimated empirical formula for PSD error to a
broader audience.


Thanks!


Allan

**Comment by haeginh** [comment, marked as answer]
Sorry for the late reply.
The formula is 15*(R^1.6). So in the case of the 5%, it would be 12.4%.
We are still updating the methodology, and hopefully it will be
incorporated soon.

Thank you,
Haegin

> Thanks for the response and information, Haegin. Choonsik had forwarded
> your response to this question to me earlier in the day as well. He wanted
> me to add the question to the NCI Forum so that others could see it as
> well.
>
> One more detailed question for you: in your response to Choonsik you
> mentioned an empirical relationship between the PSD error reported by
> NCIRF
> and the actual error that would result for the PSD output values.
>
> Let’s say the PSD error according to the NCIRF output is 5%, or 0.05. Does
> that mean the actual PSD value is roughly overestimated by (15*0.05)^1.6 =
> 0.63 or 63%? Am I understanding how you set up that formula correctly?
>
>
> I did not want to put this question on the NCI Forum in case you all were
> not ready to disclose the estimated empirical formula for PSD error to a
> broader audience.
>
>
> Thanks!
>
>
> Allan
>
>

**Comment by mathomas1481** [comment]
Thank you, Haegin. This makes a bit more sense in terms of the actual PSD output error. Thanks for clarifying.

Allan
